services/provider/client/client.go

```diff
@@ -2,49 +2,60 @@ package client
 
 import (
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/red-hat-storage/ocs-operator/services/provider/common"
 	pb "github.com/red-hat-storage/ocs-operator/services/provider/pb"
 	"google.golang.org/grpc"
-	"k8s.io/klog"
 )
 
-type ConsumerClient struct {
+type OCSProviderClient struct {
```

Comment:
Why the rename?  The provider Client is a consumer.
I would have services/provider/server.go and services/consumer/client.go
another option is services/provider/provider.go and services/provider/consumer.go